tests/bluetooth/bsim_bt/edtt_ble_test_app/gatt_test_app/src/gatt/gatt_macs.h

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright (c) 2019 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @brief GATT Macros
+ *
+ *  This code is auto-generated from the Excel Workbook
+ *  'GATT_Qualification_Test_Databases.xlsm' Sheet: 'Large Database 1'
+ *
+ *  Feel free to change it - but be aware that your changes might be
+ *  overwritten at the next generation...
+ */
+
+#ifndef GATT_MACS_H
+#define GATT_MACS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <bluetooth/gatt.h>
+
+/** @def BT_GATT_ATTRIBUTE
+ *  @brief Attribute Declaration Macro.
+ *
+ *  Helper macro to declare an attribute.
+ *
+ *  @param _uuid   Attribute uuid.
+ *  @param _perm   Attribute access permissions.
+ *  @param _read   Attribute read callback.
+ *  @param _write  Attribute write callback.
+ *  @param _value  Attribute value.
+ *  @param _handle Attribute handle.
+ */
+#undef  BT_GATT_ATTRIBUTE
+#define BT_GATT_ATTRIBUTE(_uuid, _perm, _read, _write, _value, _handle) \
```

Comment:
It feels a bit strange to redefine all these macros. Maybe choose a different name?

Reply:
Renamed all these instead of redefining